app/page.tsx

```diff
@@ -43,10 +45,33 @@ const ITEMS_PER_PAGE = 12;
 const SEARCH_DEBOUNCE_DELAY = 600;
 const DEFAULT_FILTER_OPTION = filterOptions[0];
 
+type CategoryOptions =
+  | ""
+  | "politics"
+  | "crypto"
+  | "technology"
+  | "sports"
+  | "international"
+  | "business"
+  | "entertainment"
+  | "economy";
+
+const categories: CategoryOptions[] = [
+  "technology",
+  "crypto",
+  "politics",
+  "economy",
+  "business",
+  "international",
+  "sports",
+  "entertainment",
+];
```

Comment:
Maybe we can use an `enum` here as both iterable object and type

```suggestion
enum CATEGORIES {
  BUSINESS = "business",
  CRYPTO = "crypto",
  ECONOMY = "economy",
  ENTERTAINMENT = "entertainment",
  INTERNATIONAL = "international",
  POLITICS = "politics",
  SPORTS = "sports",
  TECHNOLOGY = "technology",
}
```

Reply:
YES, great! Thanks